Source/player.cpp

```diff
@@ -2880,12 +2880,14 @@ void RestartTownLvl(Player &player)
 	player.setLevel(0);
 	player._pInvincible = false;
 
-	SetPlayerHitPoints(player, 64);

```

Comment:
Won't this potentially have someone spawn with more then 1 HP?

Reply:
The current way? or the new way?

Reply:
Actually I think it's a good idea to enforce it always setting to 1 no matter what.